docs/getting_started.md

```diff
@@ -0,0 +1,86 @@
+
+## Video
+
+Norfair offers optional functionality to process videos (mp4 and mov formats are supported) or capture a live feed from a camera.
+To use this functionality you need to install Norfair with the `video` extra using this command: `pip install norfair[video]`.
+
+Check the [Video class][norfair.video.Video] for more info on how to use it.
+
+## Tracking
+
+Let's dive right into a simple example in the following snippet:
+
+``` python
+from norfair import Detection, Tracker, Video, draw_tracked_objects
+
+detector = MyDetector()  # Set up a detector
+video = Video(input_path="video.mp4")
+tracker = Tracker(distance_function="frobenious", distance_threshold=100)
+
+for frame in video:
+   detections = detector(frame)
+   norfair_detections = [Detection(points) for points in detections]
+   tracked_objects = tracker.update(detections=norfair_detections)
+   draw_tracked_objects(frame, tracked_objects)
+   video.write(frame)
+```
+
+The tracker is created and then the detections are fed to it one frame at a time in order. This method is called _online tracking_ and allows Norfair to be used in live feeds and real-time scenarios where future frames are not available.
+
+Norfair includes functionality for creating an output video with drawings which is useful for evaluating and debugging. We usually start with this simple setup and move from there.
+
+### Next Steps
+
+The next steps depend a lot on your goal and the result of evaluating the output videos, nevertheless here are some pointers that might help you solve common problems
+
+#### Detection Issues
+
+Most common problem is that the tracking has errors or is not precise enough. In this case, the first thing to check is whether this is a detection error or a tracking error. As mentioned above if the detector fails the tracking will suffer.
+
+To debug this use [`draw_points`][norfair.drawing.draw_points] or [`draw_boxes`][norfair.drawing.draw_boxes] to inspect the detections and analyze if they are precise enough. If you are filtering the detections based on scores, this is a good time to tweak the threshold. If you decide that the detections are not good enough you can try a different architecture, a bigger version of the model, or consider fine-tuning the model on your domain.
+
+
+#### Tracking Issues
+
+After inspecting the detections you might find issues with the tracking, several things can go wrong with tracking but here is a list of common errors and things to try:
+
+- Objects take **too long to start**, this can have multiple causes:
+    - `initialization_delay` is too big on the Tracker. Makes the TrackedObject stay on initializing for too long, `3` is usually a good value to start with.
+    - `distance_threshold` is too big on the Tracker. Prevents the Detections to be matched with the correct TrackedObject. The best value depends on the distance used.
```

Comment:
Didn't get the idea behind this recommendation. Shouldn't it be that the `distance_threshold` is too **small**?